src/AutorestSwift/View Models/ReturnTypeViewModel.swift

```diff
@@ -38,21 +38,25 @@ struct ReturnTypeViewModel {
     init(from responses: [ResponseViewModel]?) {
         var objectTypes = Set<String>()
         var strategies = Set<String>()
+        var isNullables = Set<Bool>()
```

Comment:
What is the purpose of this? If you are trying to align objectType, strategy and nullability using an index, I don't think that works.

Reply:
Thanks. Removed.